libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/OBMySQLSchemaAccessor.java

```diff
@@ -121,6 +123,23 @@ public List<DBObjectIdentity> listTables(String schemaName, String tableNameLike
         return results;
     }
 
+    @Override
+    public List<DBUser> detailUsers(List<String> usernames) {
```

Comment:
query user detail by usernames list is not a common scenario.

what if listAllUsers ?

Reply:
this method `detailUsers` has placed in `OscDBAccessor` which is osc service code

Reply:
This PR will be close, query user account lock status will put in osc service